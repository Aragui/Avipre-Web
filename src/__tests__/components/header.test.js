import {screen, render} from '@testing-library/react'
import Header from '../../components/header'
import '@testing-library/jest-dom'

describe('<Header />', () => {
    const {container, get} = render(<Header/>)
    test('Render Header', () =>{
        expect(container).toMatchSnapshot();
    });
})