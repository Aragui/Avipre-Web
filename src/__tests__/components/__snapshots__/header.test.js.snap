// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Render Header 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg sticky-top"
  >
    <div
      class="container"
    >
      <a
        href="/"
      >
        <img
          alt=""
          class="navbar-brand"
          src="./images/logos/logotipo-2322x688-13.png"
          style="height: 70px;"
        />
      </a>
      <button
        aria-controls="navbarToggler"
        aria-expanded="false"
        aria-label="Abrir navegación"
        class="navbar-toggler"
        data-bs-target="#navbarToggler"
        data-bs-toggle="collapse"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="collapse navbar-collapse"
        id="navbarToggler"
      >
        <ul
          class="navbar-nav me-auto mb-2 mb-lg-0"
        >
          <li
            class="nav-item"
          >
            <a
              class="menu-link nav-link"
              href="/"
            >
              Inicio
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="menu-link nav-link"
              href="/#proyectos"
            >
              Proyectos
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="menu-link nav-link"
              href="/#maquinaria"
            >
              Maquinaria
            </a>
          </li>
          <li
            class="nav-item"
          >
            <a
              class="menu-link nav-link"
              href="/#ubicacion"
            >
              Ubicación
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
</div>
`;
